### PR TITLE
[RW-3988][Risk=no] Srubenst/preview data time formatting

### PR DESCRIPTION
**Pull request by blrubenstein**
During the workshop, we realized all the dates in the preview table in dataset builder were very close to the Linux Epoch. When I poked around locally, the values from the API seemed to be similarly incorrect. When looking at the values side by side with something that would be pointing to a date similar to what we might expect, it looked like they might be off by a factor of 1000. Ariel and I determined BQ was likely returning data in seconds rather than milliseconds.

Tested manually and it looks fine

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/api/DataSetController.java [hunk 1]
```diff
@@ -422,7 +422,8 @@ private void formatTimestampValues(List<DataSetPreviewValueList> valuePreviewLis
               value -> {
                 if (!value.equals(EMPTY_CELL_MARKER)) {
                   Double fieldValue = Double.parseDouble(value);
-                  queryValues.add(dateFormat.format(new Date(fieldValue.longValue())));
+                  // Bigquery returns time values in seconds, rather than in milliseconds.
+                  queryValues.add(dateFormat.format(new Date(fieldValue.longValue() * 1000)));
```
Comment (jaycarlton) [inline api/src/main/java/org/pmiops/workbench/api/DataSetController.java]: You shouldn't have to do conversions inline here. There are things like [ofEpochSecond](https://docs.oracle.com/javase/8/docs/api/java/time/LocalDateTime.html#ofEpochSecond-long-int-java.time.ZoneOffset-)

Comment (jaycarlton) [reply api/src/main/java/org/pmiops/workbench/api/DataSetController.java]: or use [Instant](https://www.tutorialspoint.com/javatime/javatime_instant_ofepochsecond.htm)

**Comment by blrubenstein** [comment]
This is the only place where that enum is checked against.

**Comment by jaycarlton** [comment]
Sounds good. Eventually we should move this closer to a BigQueryUtils or something. Maybe have a method that takes in a field and a DateFormatter to make a string.

**Pull request merged**